scss-test.scss

```diff
@@ -0,0 +1,19 @@
+.unsorted {
+  color: #FFFFFF;
+  background: red;
+  .inception-1 {
+    color: #DDD;
+    .inception-2 {
+       color: #ccc;
+       inception-3 {
+         color: #bbb;
+          inception-4 {
+            color: #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.